.github/workflows/ci.yml

```diff
@@ -268,7 +268,7 @@ jobs:
     - name: set up python
       uses: actions/setup-python@v4
       with:
-        python-version: '3.10'
+        python-version: '3.11'
```

Comment:
I've updated this because of [failure on Python 3.10](https://github.com/pydantic/pydantic/actions/runs/6343083711/job/17230801075?pr=7696)